{{cookiecutter.project_slug}}/config/settings/common.py

```diff
@@ -53,9 +53,10 @@
 
 # MIDDLEWARE CONFIGURATION
 # ------------------------------------------------------------------------------
-MIDDLEWARE_CLASSES = (
+MIDDLEWARE = (
     'django.middleware.security.SecurityMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.locale.LocaleMiddleware',
```

Comment:
I know we should include it, but is `LocaleMiddleware` a requirement for Django 1.10?